tests/features/metadata/update.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Update metadata
+    User has deployed RSTUF,
+    User has run the ceremony and completed bootstrap successfully,
+
+    Scenario: Updating Root metadata full signed
+        Given RSTUF is running and operational
+        Then the RSTUF is receiving multiple requests
```

Comment:
nit: Would this also make sense?

```suggestion
        Given RSTUF is receiving multiple requests
```

AFAICS in test_update.py, the `given`-decorated function currently does not do anything. So, why not use the add artifact requests as context of your test. 🤷

Reply:
Besides, we don't do anything related to "running and operational" in the BDD tests. We perform it using the scripts, where we do the deployment, ceremony, and bootstrap, which is required before receiving artifacts.
I tried to be explicit in the given requirements to evaluate the behavior.